VAMobile/src/translations/en/common.json

```
@@ -75,6 +75,17 @@
   "goToMyVAHealth": "Go to My VA Health",
   "goToMyVAHealth.a11yLabel": "Go to My V-A Health",
   "howToAttachAFile": "How to attach a file",
+  "inAppRecruitment.contracts": "VA contracts: 36C10B22C0011 & 36C10X18C0061",
+  "inAppRecruitment.giveFeedback": "Give feedback",
+  "inAppRecruitment.giveFeedback.a11yHint": "Go to the give feedback page",
+  "inAppRecruitment.goToQuestionnaire": "Go to questionnaire",
+  "inAppRecruitment.goToQuestionnaire.loading": "Loading questionnaire...",
+  "inAppRecruitment.learnMore": "Learn more about the Veteran Usability Project",
+  "inAppRecruitment.makeAppBetter.body": "You can help us improve this app by being part of online feedback sessions.",
+  "inAppRecruitment.makeAppBetter.header": "Make this app better for all Veterans",
+  "inAppRecruitment.makeAppBetter.bullet.1": "Register once by completing a 5 minute questionnaire.",
```

```suggestion
  "inAppRecruitment.makeAppBetter.bullet.1": "Register once by completing a 5-minute questionnaire.",
```
Based on the Figma mock

---

fixed